mkosi/distributions/debian.py

```diff
@@ -178,6 +178,8 @@ def setup_apt(state: MkosiState, repos: Sequence[str]) -> None:
             APT::Install-Recommends "false";
             APT::Get::Assume-Yes "true";
             APT::Get::AutomaticRemove "true";
+            APT::Get::allow-change-held-packages "true";
+            APT::Get::allow-remove-essential "true";
```

Comment:
Please capitalize the individual words to be consistent with the rest of the settings we apply

Reply:
which words? apt specifies these options in lower-case:
```
../apt$ grep -Erni '(allow-change-held|allow-remove-essential)' apt-private/
apt-private/private-cmndline.cc:275:   addArg(0,"allow-change-held-packages","APT::Get::allow-change-held-packages",CommandLine::Boolean);
apt-private/private-cmndline.cc:276:   addArg(0,"allow-remove-essential","APT::Get::allow-remove-essential",CommandLine::Boolean);
apt-private/private-install.cc:223:      if (Essential == true && _config->FindB("APT::Get::allow-remove-essential", false) == false)
apt-private/private-install.cc:224:         return _error->Error(_("Essential packages were removed and -y was used without --allow-remove-essential."));
apt-private/private-install.cc:227:      if (Hold == true && _config->FindB("APT::Get::allow-change-held-packages", false) == false)
apt-private/private-install.cc:228:         return _error->Error(_("Held packages were changed and -y was used without --allow-change-held-packages."));
apt-private/private-install.cc:302:   if (Essential == true && Safety == true && _config->FindB("APT::Get::allow-remove-essential", false) == false)

```

Reply:
From the apt conf manpage:

> Case is not significant in names of configuration items, so in the previous example you could use dpkg::pre-install-pkgs.

So please capitalize them like we do the rest of the options, case doesn't matter to apt.

Reply:
Done. Note there are 4 other config items in that list that aren't capitalised.